installation/synology.md

```diff
@@ -32,7 +28,7 @@ The filename and format of the uploaded file will be checked before installation
 If there is not a Java8 package in the Package Center, your machine may not be supported by the [Synology package](https://www.synology.com/en-us/dsm/packages/Java8).
 
 The best option for older unsupported Synology models such as **PowerPC** and **ARJava SEM** is to include the *PC-Loadletter* Repo.
-[Download Java SE](http://tinyurl.com/javaembed) (Java Cryptography Extension if required) and put them in the *public* folder of your NAS.
+[Download Java SE](http://tinyurl.com/javaembed) [(latest ARMv6)](http://www.oracle.com/technetwork/java/embedded/embedded-se/downloads/javase-embedded-downloads-2209751.html) [(Java Cryptography Extension if required)](http://www.oracle.com/technetwork/java/javase/downloads/jce8-download-2133166.html) and put them in the *public* folder of your NAS.
```

Comment:
Since the user won't be seeing the url, can you change the link from tinyurl to where it points to: i.e. "http://www.oracle.com/technetwork/java/embedded/embedded-se/downloads/index.html". 

IMO redirect links are fairly hazardous.